lib/grape-rabl/formatter.rb

```diff
@@ -29,12 +29,17 @@ def render
             end
           end
         else
-          Grape::Formatter::Json.call object, env
+          fallback_formatter.call object, env
         end
       end
 
       private
 
+      def fallback_formatter
+        return @fallback_formatter if defined?(@fallback_formatter)
+        @fallback_formatter = Grape::Formatter.formatter_for(env['api.format']) || Grape::Formatter::Json
```

Comment:
1) Is there a reason to do the whole return and check? `@fallack_formatter ||= ...` will do just as well, simpler. 
2) It's only used once, do we need to retain a reference to it? I think this method can just return without memoizing the formatter.

Reply:
If it's only used once then definitely not. I'll get rid of the memoization. (It was initially three lines which made the early return more useful)